docs/en/community/contributing.md

```diff
@@ -142,15 +143,15 @@ This will allow you to use the `git push` command to push code directly next tim
 
 <img src="https://user-images.githubusercontent.com/27466624/204302289-d1e54901-8f27-4934-923f-fda800ff9851.png" width="1200">
 
-(2) Modify the PR description according to the guidelines so that other developers can better understand your changes
+(2) Modify the PR description according to the guidelines so that other developers can better understand your changes(note the button *base* on the left side of the PR branch which is needed to modified to dev branch)
 
-<img src="https://user-images.githubusercontent.com/27466624/204303311-84456397-ee41-44f9-945c-85ce415da235.png" width="1200">
+![204303311-84456397-ee41-44f9-945c-85ce415da235](https://i.imgur.com/1CmhbOQ.jpg)
```

Comment:
Upload images to `githubusercontent` and use the previous format(`<img src="..." width="1200">`)